src/main/kotlin/tanvd/grazi/grammar/LangToolFixes.kt

```diff
@@ -0,0 +1,24 @@
+package tanvd.grazi.grammar
```

Comment:
Move it to `tanvd.grazi.language` package

Reply:
Done